ledger-core/v2/network/consensus/gcpv2/phasebundle/ph01ctl/phase01.go

```diff
@@ -160,9 +163,11 @@ func (c *Phase01Controller) workerPhase01(ctx context.Context) {
 
 	var nsh proofs.NodeStateHash
 	startIndex := 0
+	phase01StartedAt := time.Now()
 
 	if ok, nshChannel := c.R.PreparePulseChange(); ok {
 		nsh, startIndex = c.workerSendPhase0(ctx, nodes, nshChannel)
+		stats.Record(ctx, metrics.Phase01Time.M(float64(time.Since(phase01StartedAt).Nanoseconds())/1e6))
```

Comment:
1. please use named constants
2. this is time critical code. Is there a guarantee that stats.Record will have low latency?

Reply:
1. why don't we just measure nanoseconds
2. we want to measure time critical code, measurements take time. what guarantees do we need to say that stats.Record has low latency? Benchmark?

Reply:
done